bpython/curtsiesfrontend/manual_readline.py

```diff
@@ -212,7 +212,7 @@ def back_word(cursor_offset, line):
     return (last_word_pos(line[:cursor_offset]), line)
 
 
-@edit_keys.on('<PADDELETE>')
+@edit_keys.on('<DELETE>')
```

Comment:
@sebastinas Right now in curtsies there isn't a PADDELETE, but it wouldn't hurt to add it if paddelete should do this too.